src/components/popover/tab/mouse/mouse-tab.component.html

```diff
@@ -84,15 +84,25 @@
     <div *ngSwitchCase="3" class="mouse__config mouse__config--speed text-center">
         <div class="help-text--mouse-speed text-left">
             <p>Press this key along with mouse movement/scrolling to accelerate/decelerate the speed of the action.</p>
-            <p>You can set the multiplier in <a href="#" title="link to the setting">link to setting</a>.</p>
         </div>
         <div class="btn-group btn-group-lg" role="group">
-            <button type="button" class="btn btn-default"
+            <div class="btn btn-default"
                 [class.btn-primary]="mouseActionParam === MouseActionParam.decelerate"
-                (click)="setMouseActionParam(MouseActionParam.decelerate)">-</button>
-            <button type="button" class="btn btn-default"
+                (click)="setMouseActionParam(MouseActionParam.decelerate)"
+            >
+                -
+                <span>Decelerate</span>
+            </div>
+            <div class="btn btn-default"
                 [class.btn-primary]="mouseActionParam === MouseActionParam.accelerate"
-                (click)="setMouseActionParam(MouseActionParam.accelerate)">+</button>
+                (click)="setMouseActionParam(MouseActionParam.accelerate)"
+            >
+                +
+                <span>Accelerate</span>
+            </div>
+        </div>
+        <div class="help-text--mouse-speed last-help  text-left">
+            <p>You can set the multiplier in the <a [routerLink]="['/settings']" title="settings">Settings</a>.</p>
```

Comment:
This should be the fix I mentioned: `title="Settings">settings`
Title with capital S, but the text is lower s.